schema-registry/rest-service/src/test/java/com/hortonworks/registries/schemaregistry/avro/AbstractAvroSchemaRegistryCientTest.java

```diff
@@ -44,20 +44,28 @@
  */
 public abstract class AbstractAvroSchemaRegistryCientTest {
 
-    @ClassRule
-    public static final DropwizardAppRule<RegistryConfiguration> DROPWIZARD_APP_RULE
-            = new DropwizardAppRule<>(RegistryApplication.class, ResourceHelpers.resourceFilePath("schema-registry-test.yaml"));
-
-    protected final String rootUrl = String.format("http://localhost:%d/api/v1", DROPWIZARD_APP_RULE.getLocalPort());
-    protected final Map<String, String> SCHEMA_REGISTRY_CLIENT_CONF = Collections.singletonMap(SchemaRegistryClient.Configuration.SCHEMA_REGISTRY_URL.name(), rootUrl);
+    protected static LocalSchemaRegistryServer localSchemaRegistryServer;
```

Comment:
This class should not really be abstract class now as it has static methods which are shared with derived classes. It is better to use composition than extending this class. You can remove static fields and use this class instance wherever this class is extended and respective methods can be used with that instance.